docs/dataset_integration_guide.md

```diff
@@ -19,32 +19,12 @@ There are three core base classes to subclass.
 
 ## Integration steps
 
-### 1. Set up template
+### 1. Initialize a new dataset
 
-#### Copy template
-From a terminal in the root of this repository, run these commands to copy a template of dataset integration code and tests.
 ```bash
-DATASET_ID="<provider>-<model>-<variant>"
-DATASET_PATH="<provider>/<model>/<variant>"
-mkdir -p src/reformatters/$DATASET_PATH
-cp -r src/reformatters/example/* src/reformatters/$DATASET_PATH
-mkdir -p tests/$DATASET_PATH
-cp -r tests/example/* tests/$DATASET_PATH
+uv run main initialize-new-dataset <provider> <model> <variant>
 ```
-
-#### Rename
-Run the following find/replaces. Follow PEP 8 on abbreviation capitalization in class names, e.g. `NoaaGfsForecastDataset`.
-
-Find and replace within `src/reformatters/$DATASET_PATH` and `tests/$DATASET_PATH`:
-1. `ExampleDataset` -> `<Producer><Model><Variant>Dataset`
-1. `ExampleTemplateConfig` -> `<Producer><Model><Variant>TemplateConfig`
-1. `ExampleRegionJob` -> `<Producer><Model><Variant>RegionJob`
-1. `ExampleDataVar` -> `<Producer>[<model>]DataVar`
-1. `ExampleInternalAttrs` -> `<Producer>[<model>]DataVar`
-1. `ExampleSourceFileCoord` -> `<Producer><Model>SourceFileCoord`
-1. `reformatters.example` -> `reformatters.<producer>.<model>.<variant>` (imports in tests)
-
-DataVar, InternalAttrs, and SourceFileCoord definitions can often be shared among mutiple datasets from the same producer.
```

Comment:
@aldenks I thought about keeping this line to keep sharing the context re why some names include variant and others do not. But because these classes are defined within the variant directory on initialization, it seemed like it might overly complicate the matter.